# Mini-COES "Think tanks Chilenos en tiempos turbulentos (2019-2023)"
**Proyecto FONDECYT: minicoesfondap-1523A0005**

## Introducción
Este informe presenta una descripción detallada de la base de datos recopilada como parte del proyecto *Mini-COES "Think tanks Chilenos en tiempos turbulentos (2019-2023)"*.
El propósito de este documento es caracterizar de manera objetiva la base de datos, describiendo sus variables, estructura, y proceso de creación.


In [1]:
import pandas as pd

file_path = r'C:\Users\Rodrigo\Desktop\MiniCOES\minicoesfondap-1523A0005\Database\01_database_think_tanks_updated.xlsx'
df = pd.read_excel(file_path)

df.head()

ID Think Tank Tipo de Think Tank Orientación Política             Autor  \
0   1        LyD        Transversal              Derecha  Francisco Garcés   
1   2        LyD        Transversal              Derecha  Francisco Garcés   
2   3        LyD        Transversal              Derecha  Francisco Garcés   
3   4        LyD        Transversal              Derecha  Francisco Garcés   
4   5        LyD        Transversal              Derecha     Ricardo Matte   

                                              Título Fecha de Publicación  \
0  India: Necesidad de Reformas Estructurales par...  2011-01-01 00:00:00   
1  Bolivia: Turbulencia Política Incesante, Auge ...  2011-01-01 00:00:00   
2  México: Potencial Económico Continúa a la Espe...  2011-01-01 00:00:00   
3            Panamá: Reformas Económicas y Expansión  2011-01-01 00:00:00   
4  Índice de Libertad Económica: Señales de Alert...  2011-01-01 00:00:00   

  Medio de Comunicación Corpus             Producto  \
0                   NaN    NaN  Coyuntura Económica   
1                   NaN    NaN  Coyuntura Económica   
2                   NaN    NaN  Coyuntura Económica   
3                   NaN    NaN  Coyuntura Económica   
4                   NaN    NaN  Coyuntura Económica   

                          Enlace  
0  Wayback Machine (archive.org)  
1  Wayback Machine (archive.org)  
2  Wayback Machine (archive.org)  
3  Wayback Machine (archive.org)  
4  Wayback Machine (archive.org)

## Descripción General de la Base de Datos

### Alcance
- **Centros de Estudio Incluidos**:
  - **Izquierda**
    - Transversales:
      - Fundación Sol
      - Nodo XXI
      - Chile 21
      - Espacio Público
      - Centro de Estudios del Desarrollo (CED)
      - Centro de Estudios Nacionales de Desarrollo Alternativo (CENDA)
      - Horizonte Ciudadano
    - De partido:
      - Rumbo Colectivo
      - Instituto Igualdad
      - Centro Democracia y Comunidad
      - Instituto de Ciencias Alejandro Lipschutz (ICAL)
      - Observatorio de Políticas Económicas (OPES)
    - Semi Difuntos:
      - Corporación de Estudios para Latinoamérica (CIEPLAN)
      - La Casa Común
      - Saberes Colectivos
  - **Derecha**
    - Transversales:
      - Libertad y Desarrollo (LyD)
      - Centro de Estudios Públicos (CEP)
      - Pivotes
      - Instituto de Estudios de la Sociedad (IES)
      - IdeaPaís
      - Instituto ResPública
      - Fundación para el Progreso (FPP)
    - De partido:
      - Fundación Jaime Guzmán
      - Horizontal
      - Ideas Republicanas
      - Instituto Libertad
    - Universidades:
      - Faro UDD
      - Signos UAndes
      - Centro Latinoamericano de Políticas Económicas y Sociales (Clapes UC)




In [2]:
df.shape

(28889, 11)

- **Número de Registros**: 28889
- **Periodo de Cobertura**: Todas las publicaciones históricas.
- **Origen de Datos**: Publicaciones reportadas en su página web.


## Estructura de la Base de Datos

La base de datos contiene las siguientes variables principales:

| **Variable**             | **Descripción**                                                                  | **Tipo de Dato**    |
|--------------------------|----------------------------------------------------------------------------------|---------------------|
| **ID**                   | Identificador único para cada registro en la base de datos.                      | Numérico (Ordinal)  |
| **Think Tank**           | Nombre del centro de estudios asociado a cada registro.                         | Categórico (Texto)  |
| **Tipo de Think Tank**   | Clasificación según su naturaleza: partido, transversal, o universidad.          | Categórico (Texto)  |
| **Orientación Política** | Clasificación ideológica del think tank (Derecha o Izquierda).                   | Categórico (Texto)  |
| **Autor**                | Nombre del autor o autores de la publicación (si está disponible).               | Texto               |
| **Título**               | Título de la publicación.                                                        | Texto               |
| **Fecha de Publicación** | Fecha en la que se realizó la publicación.                                       | Temporal (Fecha)    |
| **Medio de Comunicación**| Medio donde se publicó la intervención (si aplica).                              | Categórico (Texto)  |
| **Corpus**               | Contenido del documento publicado.                                               | Texto               |
| **Producto**             | Tipo de documento publicado: Carta al Director, Columna, Documento Académico, etc.| Categórico (Texto)  |
| **Enlace**               | Enlace al recurso digital de la publicación.                                     | Texto (URL)         |


In [3]:
df.dtypes

ID                        int64
Think Tank               object
Tipo de Think Tank       object
Orientación Política     object
Autor                    object
Título                   object
Fecha de Publicación     object
Medio de Comunicación    object
Corpus                   object
Producto                 object
Enlace                   object
dtype: object


## Cobertura
- **Periodo Cubierto**: La base de datos abarca intervenciones realizadas entre octubre de 2019 y diciembre de 2023. Los períodos clave incluyen:
  - **18 de Octubre de 2019**: Inicio del estallido social.
  - **15 de Noviembre de 2019**: Acuerdo por la Paz.
  - **18 de Marzo de 2020**: Decreto de Estado de Excepción por la pandemia de COVID-19.
  - **27 de Octubre de 2020**: Plebiscito de Entrada.
  - **16 de Mayo de 2021**: Elección de Convencionales Constituyentes.
  - **4 de Julio de 2021**: Inicio de la Convención Constitucional.
  - **18 de Octubre de 2021**: Inicio de la Discusión Temática en la Convención.
  - **15 de Febrero de 2022**: Inicio de los Plenos en la Convención.
  - **29 de Abril de 2022**: Inicio de las Comisiones Finales (Armonización, Preámbulo, Normas Transitorias).
  - **4 de Julio de 2022**: Entrega oficial del proyecto constitucional.
  - **4 de Septiembre de 2022**: Plebiscito de Salida.
  - **12 de Diciembre de 2022**: Publicación del Documento de Bases para el nuevo proceso constituyente.
  - **24 de Enero de 2023**: Lista de 24 expertos designados para el proceso.
  - **6 de Marzo de 2023**: Inicio de la Comisión Experta.
  - **7 de Mayo de 2023**: Elección de Consejeros Constitucionales.
  - **30 de Mayo de 2023**: Fin del Anteproyecto Constitucional.
  - **7 de Junio de 2023**: Inicio de sesiones del Consejo Constitucional.
  - **7 de Noviembre de 2023**: Cierre del Consejo Constitucional.
  - **17 de Diciembre de 2023**: Plebiscito de Salida del segundo proceso constituyente.



### Representatividad
- **Centros de Estudio Representados**: 
  - **Izquierda**: La base de datos incluye **19 centros de estudio** asociados a orientaciones progresistas o de izquierda:
    - **Transversales (7)**: Fundación Sol, Nodo XXI, Chile 21, Espacio Público, Centro de Estudios del Desarrollo (CED), Centro de Estudios Nacionales de Desarrollo Alternativo (CENDA), Horizonte Ciudadano.
    - **De partido (6)**: Rumbo Colectivo, Instituto Igualdad, Centro Democracia y Comunidad, Instituto de Ciencias Alejandro Lipschutz (ICAL), Observatorio de Políticas Económicas (OPES).
    - **Semi Difuntos (3)**: Corporación de Estudios para Latinoamérica (CIEPLAN), La Casa Común, Saberes Colectivos.
  - **Derecha**: La base de datos incluye **16 centros de estudio** asociados a orientaciones conservadoras o de derecha:
    - **Transversales (7)**: Libertad y Desarrollo (LyD), Centro de Estudios Públicos (CEP), Pivotes, Instituto de Estudios de la Sociedad (IES), IdeaPaís, Instituto ResPública, Fundación para el Progreso (FPP).
    - **De partido (5)**: Fundación Jaime Guzmán, Horizontal, Ideas Republicanas, Instituto Libertad.
    - **Universidades (3)**: Faro UDD, Signos UAndes, Centro Latinoamericano de Políticas Económicas y Sociales (Clapes UC).


In [4]:
import plotly.express as px

orientation_counts = df['Orientación Política'].value_counts().reset_index()
orientation_counts.columns = ['Orientación Política', 'Número de IDs']

fig = px.bar(orientation_counts, x='Orientación Política', y='Número de IDs', 
             title='Número de IDs por Orientación Política', 
             labels={'Orientación Política':'Orientación Política', 'Número de IDs':'Número de IDs'},
             color='Orientación Política', color_discrete_map={'Derecha':'red', 'Izquierda':'blue'})

fig.show()
fig = px.pie(orientation_counts, names='Orientación Política', values='Número de IDs', 
             title='Distribución de IDs por Orientación Política', 
             color='Orientación Política', color_discrete_map={'Derecha':'red', 'Izquierda':'blue'})

fig.show()

In [5]:
think_tank_counts = df['Think Tank'].value_counts().reset_index()
think_tank_counts.columns = ['Think Tank', 'Número de IDs']
think_tank_counts = think_tank_counts.sort_values(by='Número de IDs', ascending=True)

fig = px.bar(think_tank_counts, x='Número de IDs', y='Think Tank', 
             title='Número de IDs por Think Tank', 
             labels={'Think Tank':'Think Tank', 'Número de IDs':'Número de IDs'},
             orientation='h', height=800)

fig.show()


In [7]:
tipo_think_tank_counts = df['Tipo de Think Tank'].value_counts().reset_index()
tipo_think_tank_counts.columns = ['Tipo de Think Tank', 'Número de IDs']
tipo_think_tank_counts = tipo_think_tank_counts.sort_values(by='Número de IDs', ascending=False)

fig = px.bar(tipo_think_tank_counts, x='Número de IDs', y='Tipo de Think Tank', 
             title='Número de IDs por Tipo de Think Tank', 
             labels={'Tipo de Think Tank':'Tipo de Think Tank', 'Número de IDs':'Número de IDs'},
             color='Tipo de Think Tank', orientation='h')

fig.show()

In [9]:

izquierda_tipo_counts = df[df['Orientación Política'] == 'Izquierda']['Tipo de Think Tank'].value_counts().reset_index()
izquierda_tipo_counts.columns = ['Tipo de Think Tank', 'Número de IDs']
izquierda_tipo_counts = izquierda_tipo_counts.sort_values(by='Número de IDs', ascending=False)

derecha_tipo_counts = df[df['Orientación Política'] == 'Derecha']['Tipo de Think Tank'].value_counts().reset_index()
derecha_tipo_counts.columns = ['Tipo de Think Tank', 'Número de IDs']
derecha_tipo_counts = derecha_tipo_counts.sort_values(by='Número de IDs', ascending=False)

fig_izquierda = px.bar(izquierda_tipo_counts, x='Número de IDs', y='Tipo de Think Tank', 
                       title='Número de IDs por Tipo de Think Tank (Izquierda)', 
                       labels={'Tipo de Think Tank':'Tipo de Think Tank', 'Número de IDs':'Número de IDs'},
                       color='Tipo de Think Tank', orientation='h')

fig_izquierda.show()

fig_derecha = px.bar(derecha_tipo_counts, x='Número de IDs', y='Tipo de Think Tank', 
                     title='Número de IDs por Tipo de Think Tank (Derecha)', 
                     labels={'Tipo de Think Tank':'Tipo de Think Tank', 'Número de IDs':'Número de IDs'},
                     color='Tipo de Think Tank', orientation='h')

fig_derecha.show()
